roomii/app/src/main/java/com/example/roomii/MainActivity.java

```diff
@@ -18,7 +18,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         BottomNavigationView navigation = findViewById(R.id.navigation);
         navigation.setOnNavigationItemSelectedListener(this);
-        
+
+        Intent intent = new Intent(this, LoginActivity.class);
```

Comment:
the app routes to loginactivity on startup and goes to main activity upon successful login, so we can get rid of this